app/src/lib/editors/darwin.ts

```diff
@@ -113,7 +121,7 @@ async function findApplication(editor: ExternalEditor): Promise<string | null> {
  */
 export async function getAvailableEditors(): Promise<
   ReadonlyArray<IFoundEditor<ExternalEditor>>
-> {
+  > {
```

Comment:
Can you remove the two spaces here?

Reply:
Sure, I have no idea how those spaces wildly appeared there.